### PR TITLE
create event json for Calgary

### PR DESCRIPTION
**Pull request by tonytvo**
create event json for Calgary, please review

### CODE REVIEW

File: _data/events/Calgary.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "title": "GDCR17 in Calgary",
+  "url": "https://www.meetup.com/Calgary-Agile-Methods-User-Group/events/244066406/",
+  "moderators": [
+    "@coderetreat",
```
Comment (rradczewski) [inline _data/events/Calgary.json]: Why did you add the global account as a Moderator here?

Comment (tonytvo) [reply _data/events/Calgary.json]: good question, I guess ... the global account would like to get news/events for the local as well.., but I guess... that's not really need it? let me know if I need to remove that...

Comment (rradczewski) [reply _data/events/Calgary.json]: The field is mostly for interested folks to checkout who will be there and facilitate. We're happy to retweet announcements with @coderetreat, feel free to message us in #coderetreat on the [Software Crafters Slack](http://slack.softwarecrafters.org)

That being said, please only add the actual facilitators/moderators in this field ☺️👍

**Comment by tonytvo** [comment]
I removed the global codetreat from the moderators, @rradczewski please review again

**Comment by rradczewski** [comment]
Looks good! Thanks and welcome 🎉🎉
